dev_guide/projects.adoc

```diff
@@ -165,3 +165,36 @@ To delete a project using the CLI:
 ----
 $ oc delete project <project_name>
 ----
+
+[[assigning-pods-to-specific-nodes]]
+
+== Assigning Pods to Specific Nodes
```

Comment:
this feels a little out of place here; if i were looking for how to affect node placement of pods, i wouldn't look under projects, and the only tie-in really is that the default is/can be by project.

not sure what would be a better section... something about creating apps, or editing deployment configurations...